release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3349,6 +3349,71 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, Operator Lifecycle Manager (OLM) catalog source pods did not recover from node failure if the `registryPoll` field was `none`. With this release, OLM `CatalogSource` registry pods recover from cluster node failures and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-42150[*OCPBUGS-42150*])
+
+* Previously, if you created a hosted cluster by using a proxy to make the cluster reach a control plane from a compute node, the compute node would be unavailable to the cluster. With this release, the proxy settings are updated for the node so that the node can use a proxy to communicate with the control plane successfully. (link:https://issues.redhat.com/browse/OCPBUGS-42021[*OCPBUGS-42021*])
+
+* Previously, when the Operator Lifecycle Manager (OLM) evaluated a potential upgrade, it used the dynamic client list for all custom resource (CR) instances in the cluster. Clusters with a large number of CRs could experience timeouts from the apiserver and stranded upgrades. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-42017[*OCPBUGS-42017*])
+
+* Previously, the {product-title} web console *Topology* view could only display a maximum of 100 nodes. If you attempted to view more than 100 nodes, the web console would output a `Loading is taking longer than expected.` error message. With this release, the `MAX_NODES_LIMIT` parameter for the web console is set to `200` so that the web console can display a maximum of 200 nodes. (link:https://issues.redhat.com/browse/OCPBUGS-41581[*OCPBUGS-41581*]
+
+* Previously, when you configured a hosted cluster to use an identity provider (IdP) that has either an `http` or `https` endpoint, the IdP hostname did not resolve when sent through the proxy. With this release, a DNS lookup operation checks the IdP before IdP traffic is sent through a proxy, so that IdPs with hostnames can only be resolved by the data plane and verified by the Control Plane Operator (CPO). (link:https://issues.redhat.com/browse/OCPBUGS-41374[*OCPBUGS-41374*])
+
+
+* Previously, the Cloud Credential Operator (CCO) would produce an error when starting or restarting when a large number of secrets in the cluster were fetched at once. With this release, CCO fetches the secrets in batches of 100 and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41236[*OCPBUGS-41236*])
+
+//* Previously, the grace period for a node to become ready was not aligned with upstream behavior. Occasionally, the grace period caused a node to cycle between `Ready` and `Not ready` states. With this release, the issue is fixed so that the grace period does not cause a node to cycle between the two states. (link:https://issues.redhat.com/browse/OCPBUGS-39378[*OCPBUGS-39378*])
```

Comment:
This note is pending an SME ack or can it be removed?

Reply:
It's pending the removal of the red hat security label :)